src/ui/app/components/transactionBuilder.jsx

```diff
@@ -268,24 +341,76 @@ const TransactionBuilder = React.forwardRef(({ onConfirm }, ref) => {
             justifyContent="center"
             flexDirection="column"
           >
-            <Image src={IOHK} width="40px" />
-            <Box h="4" />
             <Text fontSize="sm">
-              Support the development of Nami Wallet by delegating to{' '}
+              Enter the Stake Pool ticker or ID to delegate your funds and start
+              receiving rewards. Alternatively, head to{' '}
               <Link
                 fontWeight="semibold"
-                onClick={() => window.open('https://pipool.online')}
+                onClick={() => window.open('https://pool.pm')}
               >
-                Berry Pool
-              </Link>{' '}
-              and earn approximately <b>3.7%</b> staking rewards per year.
-              Alternatively, head to https://pool.pm/, connect your Nami wallet
-              and delegate to a stake pool of your choice.
+                https://pool.pm
+              </Link>
+              , connect your Nami wallet and delegate to a stake pool of your
+              choice
             </Text>
             <Box h="6" />
-            {data.error ? (
+            <Tooltip
+              label={`${data.pool.ticker} ${data.pool.name}`}
+              placement="top"
+              isOpen={data.pool.state === PoolStates.DONE ? undefined : false}
+            >
+              <InputGroup size="md">
+                <Input
+                  variant="filled"
+                  h={8}
+                  pr={poolHasTicker(data.pool) ? '2rem' : '4.5rem'}
+                  pl={'0.5rem'}
+                  type="text"
+                  fontSize="14px"
+                  color={
+                    data.pool.state === PoolStates.DONE ? '#A3AEBE' : undefined
+                  }
+                  value={data.pool.id}
+                  onChange={(e) => {
+                    setData((s) => ({
+                      ...s,
+                      pool: {
+                        ...s.pool,
+                        id: e.target.value,
+                        state: PoolStates.EDITING,
+                      },
+                    }));
+                  }}
+                  placeholder="Enter Pool ID"
+                  onKeyDown={(e) => {
+                    if (e.key == 'Enter') prepareDelegationTx();
+                  }}
+                />
+                <InputRightElement {...poolRightElementStyle(data.pool)}>
+                  {data.pool.state === PoolStates.EDITING && (
+                    <Button
+                      h={6}
+                      size="sm"
+                      colorScheme="teal"
+                      disabled={data.pool.id === '' || data.pool.isLoading}
+                      isLoading={data.pool.isLoading}
+                      onClick={() => prepareDelegationTx()}
```

Comment:
why not `onClick={prepareDelegationTx}`?

Reply:
Can we resolve this @pczeglik-iohk ?